e2e-tests/page-objects/shared/profile-page.js

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+const util = require('./util');
+
+class ProfilePage {
+
+  constructor() {
+    this.$$claims = $$('td');
+  }
+
+  waitForPageLoad() {
+    return util.wait(this.$$claims);
+  }
+
+  logout() {
+    return this.$logoutLink.click();
+  }
+
+  containsClaim(expectedClaim) {
+    return this.$$claims.filter((claim) => {
+      return claim.getText().then((text) => {
+        if (text === expectedClaim) {
+          return true;
+        }
+      });
+    }).then(claim => {
+      return claim.length >= 1;
```

Comment:
Correct me if I'm wrong, but this is saying "Return `true` if 1 or more claims match"

Reply:
That's right.  @vijetmahabaleshwar-okta I agree that this could be simplified to look for "id='claim-email'" rather than iterate through TD elements

Reply:
Done